tools/clang/test/CodeGenHLSL/atomic2.hlsl

```diff
@@ -0,0 +1,19 @@
+// RUN: %dxc -E main -T cs_6_0 %s | FileCheck %s
+
+// CHECK: dx.op.atomicBinOp.i32(i32 78
```

Comment:
It would be better to write the check on one line with a regex for the unimportant parts to make it clear we are checking the operands of the atomicBinOp intrinsic. Something like

    CHECK: dx.op.atomicBInOp.i32(i32 78, {{.*}}, i32 0, i32 undef, i32 1)